types/three/examples/jsm/exporters/GLTFExporter.d.ts

```diff
@@ -86,9 +86,13 @@ export class GLTFExporter {
         input: Object3D | Object3D[],
         options?: GLTFExporterOptions,
     ): Promise<ArrayBuffer | { [key: string]: any }>;
+
+    static Utils: {
+        GLTFWriter: typeof GLTFWriter;
+    };
```

Comment:
I see `Utils` being set on `GLTFExporter` but I'm having trouble seeing where `GLTFWriter` is declared on it. Can you show where that happens?